pipeline/terraform/modules/es_index/main.tf

```diff
@@ -16,6 +16,7 @@ locals {
 }
 
 resource "elasticstack_elasticsearch_index" "the_index" {
+  deletion_protection  = false
```

Comment:
This should probably be omitted - I keep committing this as well (https://github.com/wellcomecollection/catalogue-pipeline/issues/2463)

Reply:
Agh I managed to avoid committing it right until my very last commit!